src/main/java/org/carservice/autoparts/AutoPartsController.java

```diff
@@ -26,4 +28,35 @@
         return service.listAll();
     }
 
+    @GetMapping("/autoparts/{id}")
+    public ResponseEntity<AutoParts> get(@PathVariable Integer id) {
+        try {
+            AutoParts autoParts = service.get(id);
```

Comment:
you can return AutoParts without wrapping it in ResponseEntity but your option works too. It's just a bit too verbose